source/renderer/app/components/staking/epochs/StakingEpochs.scss

```diff
@@ -1,5 +1,174 @@
+      padding-top: 0;
+    }
+  }
+
+  .noResultsLabel {
+    color: var(--theme-staking-font-color-lighter);
+    flex-grow: 1;
+    font-family: var(--font-regular);
+    font-size: 16px;
+    text-align: center;
+  }
+
+  .loadingSpinnerWrapper {
+    margin: auto;
+  }
+
+  .currentEpochProgressBar {
+    height: 24px;
+    margin-bottom: 31px;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-top: -2px;
+  }
+
+  .progressBarContainer {
+    background-color: var(--theme-staking-background-color);
+    border-radius: 5px;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .progress {
+    background: repeating-linear-gradient(
+      -63deg,
+      var(--theme-staking-progress-stripe-dark-1-background-color),
+      var(--theme-staking-progress-stripe-dark-1-background-color) 10px,
+      var(--theme-staking-progress-stripe-dark-2-background-color) 10px,
+      var(--theme-staking-progress-stripe-dark-2-background-color) 20px
+    );
+    border-radius: 5px;
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+  }
+
+  .overlapProgressLabel {
+    @extend %progressLabelText;
+    color: #fff;
+    transform: translateX(calc(-100% - 8px));
+    z-index: 1000;
+  }
+
+  .progressLabel {
+    @extend %progressLabelText;
+    color: var(--theme-staking-font-color-regular);
+    right: 8px;
+    z-index: 500;
+  }
+
+  table {
+    border-style: hidden;
+    user-select: text;
+    width: 100%;
+
+    thead {
+      background-color: var(--theme-staking-table-head-background-color);
+    }
+
+    tr,
+    td,
+    th {
+      border: 1px solid var(--theme-staking-table-border-color);
+    }
+
+    th {
+      color: var(--theme-staking-font-color-regular);
+      cursor: pointer;
+      font-family: var(--font-semibold);
+      font-size: 14px;
+      line-height: 1.14;
+      padding: 14px 20px;
+      text-align: left;
+
+      &:hover {
+        & > .sortIcon {
+          opacity: 0.5;
+          visibility: visible;
+        }
+      }
+
+      & > .sortIcon {
+        margin-left: 6px;
+        visibility: hidden;
+        & > svg {
+          height: 7.5px;
+          width: 7px;
+          & > path {
+            fill: var(--theme-staking-font-color-regular);
+          }
+        }
+        &.ascending {
+          & > svg {
+            transform: rotate(180deg);
+          }
+        }
+        &.sorted {
+          opacity: 1;
+          visibility: visible;
+        }
+      }
+    }
+
+    td {
+      color: var(--theme-staking-font-color-regular);
+      font-family: var(--font-light);
+      font-size: 14px;
+      line-height: 1.3;
+      padding: 14px 20px;
+      text-align: left;
+      width: 25%;
+
+      p {
+        -webkit-box-orient: vertical;
+        -webkit-line-clamp: 1;
+        display: -webkit-box;
+        overflow: hidden;
+        text-overflow: ellipsis;
+
+        .stakePoolReference {
+          color: var(--theme-staking-table-body-highlighted-text-color);
```

Comment:
I think @darko-mijic requested this to not be a link. It should be only bold, but not blue. cc @nikolaglumac @a-rukin

Reply:
I'm not sure. This can be one more place to open stake pool dialog (with delegate button)

Reply:
I remember @darko-mijic saying this is an MVP, to keep things simple. But not sure...

Reply:
No link or dialog needed here.

Reply:
@yakovkaravelov please, just removed the blue color, leaving with the same color as the normal text, but leave the bold weight. Thanks.